articles/getting_started.md

```diff
@@ -90,7 +90,8 @@ There is more information about [accounts, certification and qualified researche
 
 <img style="float: right" src="/assets/images/project_1.png" >
 
-Now that you have your Synapse account we can start creating content. All Synapse content is organized according to user-created `Projects`.  `Projects` are an organizational unit in which you can collaboratively access and share `Wikis` (narratives), `Files` (a distributed file system to store data, code, and results from your work), `Tables` (web-accessible, sharable, and queryable data where columns can have a user-specified structured schema). Each project also contains a project specific `Discussion` Forum.
+Now that you have your Synapse account, let's get started. All Synapse content is organized according to user-created `Projects`. Select a unique name for your Project, such as "My uniquely named project", and create your Project. Projects are an organizational unit in which you can collaboratively access and share `Wikis` (narratives), `Files` (a distributed file system to store data, code, and results from your work), and Tables (web-accessible, sharable, and queryable data where columns can have a user-specified structured schema). Each project also contains a project specific `Discussion Forum`.
```

Comment:
"lets get started" is awkward.
Change back to:
Now that you have your Synapse account you can start adding content.

Also update the "My uniquely named project" to use the name generator that kenny submitted?